### DISCUSSION TITLE
Detection de valeurs numériques anormales [tour_dep_transp] (3.1.26.4)

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#59] [Q&A]
@VeloTerritoires
Le chapitre 3.1.26.4 indique comment remplir les champs  [tour_dep_transp_jour_valide] et [tour_dep_transp_jour_corrige].
Par contre je n'ai pas d'indication sur la détection d'anomalies sur [tour_dep_transp]

Est-ce que Eva-scan doit juste détecter les cas pour lesquels [dist_transp_jour] et [distance_domicile_enq_reelle] sont proches ? Y a t il d'autres traitement à effectuer?

**Comment by SteffMangin** [comment]
@JMPivette 
Bon tu es mon parfais testeur pour des améliorations futures du guide.
En fait pour toutes les dépenses, il faut faire la comparaison par rapport aux valeurs médianes : "La première chose à faire
est donc de calculer les dépenses médianes pour chaque poste, sur la base des répondants déclarant réaliser une dépense pour le poste concerné. Puis, il est nécessaire de vérifier, pour chaque questionnaire, l’écart à la médiane. Si l’écart est trop important, il conviendra d’identifier si la dépense correspond plutôt à celle du groupe ou si elle est vraiment atypique. Une attention particulière devra être apportée aux réponses sur les dépenses dans le cas de réponses multiples au sein d’un même groupe."

En l'occurrence pour le transport, il faudrait faire une comparaison par mode de transport. 
Ensuite et en plus pour le transport, pour certains mode de transports, on refait un calcul car la perception des gens est assez mauvaise sur la dépense liée à ce poste.

J'espère que cela répond à ta question. N'hésite pas à appeler si ce n'est pas clair.

A+

**Reply by JMPivette** [reply]
Ok pour faire un groupement par [mode_transport_jour].
Par contre ce champs est lié à un QCM et j'ai donc de nombreuses valeurs uniques (24 dans le fichiers d'exemple) avec des combinaisons rares (exemple "Bus Urbain, Bateau" ou "Train, Bus urbain, Car").
Est-ce que je ne devrais pas garder juste les modes de transports les plus communs (> 100 réponses) et regrouper le reste dans une unique catégorie?

**Reply by SteffMangin** [reply]
@JMPivette 
tu fais la médiane pour chaque variable unique. Il y a en 8 : 
[mode_transp_jour_aucun]
[mode_transp_jour_voiture]
[mode_transp_jour_train]
[mode_transp_jour_bus]
[mode_transp_jour_autocar]
[mode_transp_jour_bateau]
[mode_transp_jour_camping_car]
[mode_transp_jour_autres]

Sachant que pour [aucun] la dépense devrait être de 0 €.

Et pour les combinaisons, je ne sais pas trop. C'est là que tu proposes de faire l'analyse sur les cas à plus de 100 réponses ?
